drivers/adxl345/adxl345.c

```diff
@@ -43,6 +44,12 @@ int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
     /* get device descriptor */
     dev->params= *params;
```

Comment:
ture, didn't notice the `*` instead of `&`, so never mind...

But still the second consideration holds: IMHO we don't need to copy all config params into RAM...